### ISSUE TITLE
[Docs] The feature dimension of 512 seems a bit small, perhaps there is a higher feature dimension

### ISSUE THREAD

**Issue by fancy-girl** [zcablii/LSKNet#58]
### Branch

master branch  https://mmrotate.readthedocs.io/en/latest/

### 📚 The doc issue

![image](https://github.com/zcablii/LSKNet/assets/28773253/512ca571-0f7d-4cf4-b7ef-d136665f0338)

1. Thankyou for your excellent work, I am using your code for a cross perspective geolocation task (which is a task that uses drones to capture images and retrieve satellite images). When I use lsk_swbackone, the output of the network structure is embed_dims=[64, 128, 320, 512], and the final output of the 512 dimensional vector seems to perform poorly. It is best for satellite retrieval drones to reach 94, while drone retrieval satellites can reach 89. 
2. I have tried adding a feature map using channel attention and adding a feature map using spatial attention, but these improvements are not significant. Experiments have shown that increasing spatial attention is indeed better than channel attention. I want to try modifying the network structure, but it may not match the pre training weights. 
3. Is there any other way to make the feature dimension higher? I think the feature dimension of papers in our field is usually 768 or 1024.

![image](https://github.com/zcablii/LSKNet/assets/28773253/75c0e390-e9c3-4414-9dcf-d6f8b61e9787)
This is my network structure, and the losses I use are cross entropy loss and triplet loss.

**Comment by zcablii** [comment]
You should not modify the network structure.  Modifying the network structure may lead to inconsistencies with the pretrained weights, potentially rendering them ineffective. Even if the modified model fits the weights perfectly, any alteration to the feature flow can disrupt the recognition of the feature space by the pretrained weights.  
If your goal is simply to increase the final channel dimension, please consider leveraging 1x1 convolution layers.
